ci/test.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash -e
 
+ENVS="py36 py37 py38 py39 py310"
+
+print_help() {
+    echo "Usage: $0 [ENV]"
+    echo ""
+    echo "  Helper script to setup and run tests (all or ENV)."
```

Comment:
We don't to run all envs in the ci, always one env. Runing all envs and skippnig missing python versions is convenience for developers so they can run "tox". The CI need more strict approach - it must specify a one env and the env must succeed.